android/src/main/java/com/swmansion/reanimated/nativeProxy/NativeProxyCommon.java

```diff
@@ -97,6 +99,35 @@ public void requestRender(AnimationFrameCallback callback) {
     mNodesManager.postOnAnimation(callback);
   }
 
+  @DoNotStrip
+  public String getReanimatedJavaVersion() {
+    return BuildConfig.REANIMATED_VERSION_JAVA;
+  }
+
+  @DoNotStrip
+  @SuppressWarnings("unused")
```

Comment:
Should we add `@SuppressWarnings("unused")` to each method that we call via JNI (in a separate PR)?

Reply:
I think yes, since there is (currently) no way of checking if a "JNI only" method is called via JNI anyway.